src/plugins/vis_default_editor/public/default_editor_controller.tsx

```diff
@@ -80,20 +78,26 @@ class DefaultEditorController {
     };
   }
 
-  render({ data, core, ...props }: EditorRenderProps) {
+  render(props: EditorRenderProps) {
     render(
-      <core.i18n.Context>
-        <KibanaContextProvider
-          services={{
-            appName: 'vis_default_editor',
-            storage: localStorage,
-            data,
-            ...core,
-          }}
+      <EuiErrorBoundary>
+        <Suspense
+          fallback={
+            <div
+              style={{
+                display: 'flex',
+                flex: '1 1 auto',
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+            >
+              <EuiLoadingChart size="xl" />
```

Comment:
We are already using the regular `EuiLoadingSpinner` in a bunch of places for lazily loading components (e.g. https://github.com/flash1293/kibana/blob/8bb67f2c6281c88cc11d95f1e30d61f6c3c8622a/src/plugins/vis_type_timeseries/public/application/components/vis_editor_lazy.tsx#L27 ) No strong opinion, but should we do the same thing here?

When this loading spinner disappears, it will show another one till the chart is loaded, right? If that's the case, I think we should show `EuiLoadingSpinner`  here.

Reply:
While chart is loading, there is no any spinner, we just have a loading bar in header.
I also don't have a strong feeling about spinner type here, I just noticed this fancy `EuiLoadingChart` in `eui` lib and craved to use it 😄

Reply:
Looks like the ping didn't work - @elastic/kibana-design

Reply:
The charts loader is fine. I can confirm there are no other loaders in Visualize or Dashboard panels. Dashboard does use the chart loading animation before the panel itself loads. I would just add `mono` to make it the same as dashboard.

```suggestion
              <EuiLoadingChart size="xl" mono/>
```

<img width="1261" alt="Screen Shot 2020-05-18 at 09 59 39 AM" src="https://user-images.githubusercontent.com/549577/82223751-12501580-98f1-11ea-911f-70ed19120583.png">